src/Common/src/Interop/Windows/mincore/Interop.DuplicateHandle.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+using System.Security;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [DllImport(Libraries.Handle, SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        [SecurityCritical]
+        internal static extern bool DuplicateHandle<TProcessHandle, TSourceHandle, TTargetHandle>(TProcessHandle hSourceProcessHandle,
```

Comment:
Turns out it the CI build actually failed, but it reported success.  Will follow-up with a separate issue.